prac_05/wimbledon.py

```diff
@@ -0,0 +1,41 @@
+"""
+Word Occurrences
+Estimate: 30 minutes
+Actual: minutes
+"""
+
+FILENAME = "wimbledon.csv"
+
+def main():
+    with open(FILENAME, "r", encoding="utf-8-sig") as in_file:
+        data = in_file.readlines()
+    champions = extract_champions(data)
+    countries = extract_countries(data)
+    country_counts = count_items(countries)
+    champion_counts = count_items(champions)
+
+    print("Wimbledon Champions:")
+    for name in champion_counts.keys():
+        print(f"{name}: {champion_counts[name]}")
+    sorted_countries = sorted(country_counts.keys())
+    print(f"These {len(sorted_countries)} countries have won the Wimbledon:")
+    print(", ".join(sorted_countries))
+
+def extract_champions(data):
+    champions = [line.strip().split(",")[2] for line in data[1:]]
+    return champions
+
+def extract_countries(data):
+    countries = [line.strip().split(",")[1] for line in data[1:]]
+    return countries
```

Comment:
This function repeats do almost the same as previous function.

Reply:
I already fixed that, thanks for your comment! ^^